### ISSUE TITLE
[document answer] @noinspection support for all plugin inspections

### ISSUE THREAD

**Issue by Doqnach** [kalessil/phpinspectionsea#190]
I've been looking around the issue tracker and search engines and could not find a divinitive answer on this.

Does the plugin support all of https://github.com/kalessil/phpinspectionsea/tree/master/src/main/resources/inspectionDescriptions for the "@noinspection" PHPDOC?

The one I'm looking to suppress is CurlSslServerSpoofingInspection in a very old legacy project (openssl is too old so need to disable all checks on validity of SSL for curl). Doesn't seem to work if I do:

```php
$this->_client = new RestClient(array(
  'base_url' => sprintf($testing ? self::BASE_URI_TEST : self::BASE_URI, self::API_VERSION),
  'username' => $user,
  'password' => $pass,
  'format_regex' => '/(\w+)\/([\w-]+(?:\+\w+)?)(;[.+])?/', // fix for hal+json
  'decoders' => array(
    'hal+json' => 'json_decode',
  ),
  'curl_options' => array(
    /** @noinspection CurlSslServerSpoofingInspection */
    CURLOPT_SSL_VERIFYHOST => 0,
    /** @noinspection CurlSslServerSpoofingInspection */
    CURLOPT_SSL_VERIFYPEER => 0,
  ),
));
```

**Comment by kalessil** [comment]
That's something not handled by IDEs correctly, unfortunately.

Possible solution, I'll add it into docs as well:
```php
$curlVersion          = curl_version();
$verifyCertificates = $curlVersion['ssl_version_number'] >= <the version where SSL will work>;
...
'curl_options' => array(
    CURLOPT_SSL_VERIFYHOST => $verifyCertificates ? 2 : 0,
    CURLOPT_SSL_VERIFYPEER  => $verifyCertificates ? 1 : 0
  )
...
```

**Comment by Doqnach** [comment]
Are there any that do work? Or is it not possible (right now) for the plugin to hook into the "@noinspection" behaviour?

**Comment by kalessil** [comment]
"@noinspection" handled by IDE we don't check the tag by ourselves.

**Comment by Doqnach** [comment]
Thanks for the clarification!

**Comment by kalessil** [comment]
No problems, I hope my hints did help)
